packages/now-cli/src/util/client.ts

```diff
@@ -33,20 +33,22 @@ export default class Client extends EventEmitter {
     forceNew = false,
     withCache = false,
     debug = false,
+    output = createOutput({ debug }),
```

Comment:
Its probably better not to create output here, right? Then all the callers need to pass in the `ctx.output`?

Reply:
Mostly addressed in https://github.com/vercel/vercel/pull/5722/commits/fe84eb790dcd49f1f010915eb93ed794a93008b7. There's probably some missing but we can address those in a follow-up. Ideally `Now` and `Client` classes would enforce an `output` being provided and remove the `debug` field in favor of `output.isDebugEnabled()`.